lexicons/app/bsky/feed/defs.json

```diff
@@ -139,6 +157,36 @@
       "properties": {
         "repost": {"type": "string", "format": "at-uri"}
       }
+    },
+    "mentionInteractionView": {
+      "type": "object",
+      "description": "The view counterpart to app.bsky.feed.post#mentionInteraction"
```

Comment:
kinndaa inclined to have these live next to one another 🤔 

but then again, we have `post` & `#postView` separate from one another 🧐

Reply:
Yeah I was a little wishy-washy about this, but that was my thinking on it.  If we were to change this, would you lean towards moving them all into `app.bsky.feed.defs` or all into `app.bsky.feed.post`?